engine/src/ui/canvas/plugin.cpp

```diff
@@ -139,7 +151,7 @@ void cubos::engine::uiCanvasPlugin(Cubos& cubos)
                     canvas.virtualSize = canvas.referenceSize;
                 }
 
-                canvas.mat = glm::ortho<float>(0, canvas.virtualSize.x, 0, canvas.virtualSize.y);
+                canvas.mat = glm::ortho<float>(0, canvas.virtualSize.x, 0, canvas.virtualSize.y, -1000.0F, 1000.0F);
```

Comment:
? Why limit it?

Reply:
Without the values set only elements in the layer 0 where being shown, I think it is probably due to the default value for zNear and zFar